src/main/src/main/java/com/shevtsov/servlets/AbstractServlet.java

```diff
@@ -0,0 +1,75 @@
+package com.shevtsov.servlets;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+
+public abstract class AbstractServlet<E> extends HttpServlet {
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+        List<E> domains = getAll();
+        try (PrintWriter printWriter = resp.getWriter()) {
+            for (E domain : domains) {
+                printWriter.println("<h5>" + domain + "<h5>");
+            }
+        }
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) {
+        try {
+            if (create(req)) {
+                return;
+            }
+        } catch (NumberFormatException e) {
+            e.printStackTrace();
+        }
+        printMsg(resp, "Creating has not been done.");
+    }
+
+    @Override
+    protected void doPut(HttpServletRequest req, HttpServletResponse resp) {
+        String id = req.getParameter("id");
+        try {
+            if (modify(req, Long.parseLong(id))) {
+                return;
+            }
+        } catch (NumberFormatException e) {
+            e.printStackTrace();
+        }
+        printMsg(resp, "Modifying has not been done.");
+    }
+
+    @Override
+    protected void doDelete(HttpServletRequest req, HttpServletResponse resp) {
+        String id = req.getParameter("id");
+        try {
+            if (remove(Long.parseLong(id))) {
```

Comment:
you can invert condition and return your message

Reply:
done